birdspotter/birdspotter/templates/index.html

```diff
@@ -54,7 +53,7 @@
                                        data-button='{"dataset_id": "{{ dataset.dataset_id }}","dataset_name": "{{ dataset.name }}"}'>Share</a>
                                 {% endif %}
                                 {% if request.user.is_authenticated %}
-                                <a class="nav-link" href="/export">Export</a>                                
```

Comment:
Don't know if this is intentional but it will eventually be changed again by actual export

Reply:
it was intentional, just to get rid of the django error that shows up when it is clicked, and yeah, I am aware of that

Reply:
This unnecessarily will cause merge conflicts with other feature branches, so I would vote against this change, as export is in progress as we speak

Reply:
fair point, just undid this change and fixed the other suggestions to the PR